src/renderer/containers/widgets/joinCommunity/joinCommunity.test.tsx

```diff
@@ -37,6 +37,7 @@ const payload = (id: string): Partial<Identity> => ({
 
 describe('User', () => {
   let socket: MockedSocket
+  let idCommunity: string
```

Comment:
It's not a big deal but in the createCommunity test you've named variable of the same kind "communityId" so it would look cleaner to keep the same naming convention